sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -95,19 +124,9 @@ export interface BeginAnalyzeHealthcareEntitiesOptions extends TextAnalyticsOper
 }
 
 // @public
-export interface BeginAnalyzeHealthcareOperationState extends AnalysisPollOperationState<PagedHealthcareEntities> {
-}
-
-// @public
-export interface BeginAnalyzeOperationState extends AnalysisPollOperationState<PagedAnalyzeResults>, AnalyzeJobMetadata {
+export interface BeginAnalyzeHealthcareOperationState extends AnalysisPollOperationState<PagedAnalyzeHealthcareEntitiesResult> {
```

Comment:
I guess JS has to start with "Begin****" in this class name. But I still feel strange.

Reply:
Dropped `Begin` for names with postfix `OperationState` because the word operation already indicates that it is an LRO and to be consistent with js form recognizer.